scripts/add_rule.py

```diff
@@ -140,7 +140,8 @@ def main(*, name: str, prefix: str, code: str, linter: str) -> None:
         variant = pascal_case(linter)
         rule = f"""rules::{linter.split(" ")[0]}::rules::{name}"""
         lines.append(
-            " " * 8
```

Comment:
that's the deviation from black we're introducing, i'm using it to confirm that it actually formats